src/main/resources/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -653,7 +653,7 @@
       "mavenCoordinates": {
          "groupId": "com.google.cloud",
          "artifactId": "google-cloud-translate",
-         "version": "1.73.0"
+         "version": "1.93.0"
```

Comment:
This one's 93 while others are 96. Intended?

Reply:
Yes, the versions are beginning to diverge.